### PR TITLE
Enable termination handler for Azure

### PR DESCRIPTION
**Pull request by alexander-demichev**
Enable termination handler for Azure

https://github.com/openshift/cluster-api-actuator-pkg/pull/170
https://github.com/openshift/cluster-api-provider-azure/pull/145

### CODE REVIEW

**Comment by alexander-demichev** [comment]
/hold 
Wait for https://github.com/openshift/cluster-api-provider-azure/pull/145

File: pkg/operator/config.go [hunk 1]
```diff
@@ -118,6 +118,8 @@ func getTerminationHandlerFromImages(platform configv1.PlatformType, images Imag
 		return images.ClusterAPIControllerAWS, nil
 	case configv1.GCPPlatformType:
 		return images.ClusterAPIControllerGCP, nil
+	case configv1.AzurePlatformType:
+		return images.ClusterAPIControllerGCP, nil
```
Comment (JoelSpeed) [inline pkg/operator/config.go:122]: Wrong image here

**Comment by enxebre** [comment]
can you please reference in the PR/commit desc the counter part PRs, e.g https://github.com/openshift/cluster-api-actuator-pkg/pull/170 and openshift/cluster-api-provider-azure#145

**Comment by enxebre** [comment]
/hold
https://github.com/openshift/machine-api-operator/pull/631#issuecomment-660935937

**Pull request merged**